.github/CODEOWNERS

```diff
@@ -865,6 +865,10 @@
 /eng/versioning/                                     @alzimmermsft @samvaity @g2vinay @JimSuplizio
 /eng/versioning/external_dependencies.txt            @alzimmermsft @samvaity @g2vinay @jonathangiles @rujche @netyyyy @saragluna @moarychan
 
+# Removing owners for these files to not require codeowner approval for changes to them given they are shared with project
+/eng/versioning/version_client.txt
```

Comment:
@JimSuplizio is the code owner linter going to complain about these entries?

Reply:
> @JimSuplizio is the code owner linter going to complain about these entries?

Apparently not in a unique manner, this isn't failing because there's already an entry missing owners and the linter is picking up that error and deduping these new errors. I'm not going to block this getting merged but I am going to have to update the parser to deal with this. I'll fix the baseline errors for this at that time.